core/vm/context/context.go

```diff
@@ -0,0 +1,142 @@
+package context
+
+import (
+	"math/big"
+
+	"github.com/celo-org/celo-blockchain/common"
+	"github.com/celo-org/celo-blockchain/consensus/istanbul"
+	"github.com/celo-org/celo-blockchain/contracts"
+	"github.com/celo-org/celo-blockchain/core/types"
+	"github.com/celo-org/celo-blockchain/core/vm"
+	"github.com/celo-org/celo-blockchain/log"
+)
+
+// New creates a new context for use in the EVM.
+func New(msg vm.Message, header *types.Header, chain vm.ChainContext, txFeeRecipient *common.Address) vm.Context {
```

Comment:
I think it makes sense to move the ChainContext and Message to this package because this is where they are used and they are not used at all in the vm package

Reply:
i moved them to the actual place they were on upstream